src/main/resources/locales/en/plugins.json

```diff
@@ -208,7 +208,28 @@
     "branch": {
       "header": "Pull Requests",
       "createPullRequest": "Create new pull request"
-    }
+    },
+    "config": {
+      "enabled": {
+        "label": "Enabled",
+        "helpText": "Enabled or disabled the branch protection"
```

Comment:
The description for this flag doesn't help at all. Maybe more like label "Enable branch protection" and helpText "Only allow write access for pull requests".